lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -27,41 +33,41 @@ def get_raw_clusters_from_folder(folder)
           end.flatten
         end
 
-        def cluster_attributes cluster, datacenter_name
+        def cluster_attributes(cluster, datacenter_name)
           {
-            :id             => managed_obj_id(cluster),
-            :name           => cluster.name,
-            :full_path      => cluster_path(cluster, datacenter_name),
-            :num_host       => cluster.summary.numHosts,
-            :num_cpu_cores  => cluster.summary.numCpuCores,
-            :overall_status => cluster.summary.overallStatus,
-            :datacenter     => datacenter_name || parent_attribute(cluster.path, :datacenter)[1],
+            id: managed_obj_id(cluster),
+            name: cluster.name,
+            full_path: cluster_path(cluster, datacenter_name),
+            num_host: cluster.summary.numHosts,
+            num_cpu_cores: cluster.summary.numCpuCores,
+            overall_status: cluster.summary.overallStatus,
+            datacenter: datacenter_name || parent_attribute(cluster.path, :datacenter)[1]
           }
         end
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{Regexp.escape(datacenter_name)}|#{Regexp.escape(datacenter.hostFolder.name)})\//,'')
+          cluster.pretty_path.gsub(%r{(#{Regexp.escape(datacenter_name)}|#{Regexp.escape(datacenter.hostFolder.name)})\/}, '')
         end
       end
 
       class Mock
-        def list_clusters(filters = { })
+        def list_clusters(*)
```

Comment:
Also, is this supported in 1.8.7? This may need to be changed back for the time being.